website/static/js/markdown.js

```diff
@@ -31,11 +32,33 @@ var bootstrapTable = function(md) {
 var oldMarkdownList = function(md) {
     md.block.ruler.after('hr', 'pyMarkdownList', pymarkdownList);
 };
+
+
+var ViewOnlyImage = function(md) {
```

Comment:
Drive-by: Use `lowerCamelCase` for consistency with the other plugins in this file.